src/custom-blocks/iframe/index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * iFrame block
+ * Frontend PHP code
+ *
+ * Uses WordPress' dynamic block method
+ * https://developer.wordpress.org/block-editor/tutorials/block-tutorial/creating-dynamic-blocks/
+ *
+ * @package mojblocks
+ *
+ */
+
+function render_callback_iFrame_block($attributes, $content)
+{
+    // Parse attributes found in index.js
+    $attribute_iFrame_url = $attributes['iFrameURL'] ?? '';
+    $attribute_iFrame_width = $attributes['iFrameWidth'] ?? '';
+    $attribute_iFrame_height = $attributes['iFrameHeight'] ?? '';
+    $attribute_iFrame_border = $attributes['iFrameBorder'] ?? false;
+    $attribute_iFrame_centre = $attributes['iFrameCentre'] ?? false;
+    $attribute_iFrame_className = $attributes['iFrameClassName'] ?? '';
+
+    if ($attribute_iFrame_url == "" || substr($attribute_iFrame_url,0,8) != "https://") {
+        return "";
+    }
```

Comment:
Do you need substr or could you do? 

```
if (empty($attribute_iFrame_url)) {
    return '';
}
$attribute_iFrame_url = esc_url($attribute_iFrame_url);
```

Reply:
https:// is needed otherwise there is a security error and it doesn't work.  So I've kept this check in.